render/id.go

```diff
@@ -14,6 +24,11 @@ func MakeEndpointID(hostID, addr, port string) string {
 	return makeID("endpoint", hostID, addr, port)
 }
 
+// MakePseudoEndpointID makes a pseudo endpoint node ID for rendered nodes.
+func MakePseudoEndpointID(hostID, addr, port string) string {
+	return makeID("pseudo-endpoint", hostID, addr, port)
+}
```

Comment:
> introduce the pseudo nodes in the process mapping instead of the endpoint mapper

That would be more consistent with how the "uncontained" pseudo nodes are handled, no?